nncf/common/tensor_statistics/statistics.py

```diff
@@ -102,11 +110,13 @@ class RawTensorStatistic(TensorStatistic):
     Base class for the raw statistics, without any aggregation.
     """
 
-    def __init__(self, values):
+    def __init__(self, values: Tensor):
         """
         :param values: Collected raw values.
         """
         self.values = values
 
-    def __eq__(self, other: "RawTensorStatistic") -> bool:
+    def __eq__(self, other: object) -> Any:
```

Comment:
```suggestion
    def __eq__(self, other: Any) -> Any:
```
Please use Any here and elsewhere